source/tank/TankFactory.hx

```diff
@@ -9,4 +11,10 @@ abstract class TankFactory {
 		tank.setControllers(new MouseFollowMoveController(tank), new SpinShootController(tank));
 		return tank;
 	}
+
+	static inline public function NewPlayerTank(X:Float = 0, Y:Float = 0) {
+		var tank = new Tank(X, Y, AssetPaths.tank_body_placeholder__png);
```

Comment:
We do eventually want the player tank to have a different graphic, though it's certainly not a requirement right at this moment.

Reply:
Should be fixed now.